doc-content/jbpm-docs/src/main/asciidoc/Persistence/persistence-configuration-file-con.adoc

```diff
@@ -0,0 +1,44 @@
+[id='persistence-configuration-file-con_{context}']
+= Configuring the `persistence.xml` file
+
+To use JPA prsistence, you must add a `persistence.xml` persistence configuration to your class path to configure JPA to use Hibernate and the H2 database (or any other database that you prefer). Place this file in the `META-INF` directory of your project.
+
+For more details about configuring the `persistence.xml` file, refer to JPA and Hibernate documentation.
```

Comment:
Need to link to the JPA and Hibernate docs if we're telling users to refer to them.

Reply:
Reference removed as I could not find a clear document, marked this in gaps list